src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -432,7 +445,7 @@ export const getL1quote = async (
                   estimatedExecutionTimeMs,
                   rate,
                   sellAmountIncludingProtocolFeesCryptoBaseUnit: sellAmountCryptoBaseUnit,
-                  buyAmountBeforeFeesCryptoBaseUnit,
+                  buyAmountBeforeFeesCryptoBaseUnit: emitAssetAmountCryptoBaseUnit,
```

Comment:
Same thing correct name, which may be indicative of this being wrong, although the memo calc will now be correct, which is the most important thing in this going in asap